### ISSUE TITLE
Linetouched event

### ISSUE THREAD

**Issue by Polymathronic** [perliedman/leaflet-routing-machine#132]
I am trying to attach a `linetouched` event to a route, but can't get it to fire, unless I set `addWaypoints` to `true`: http://jsfiddle.net/eqwscwym/8/

Is this by design? Is there another solution?

**Comment by perliedman** [comment]
Yes, this is maybe a bit backwards - `linetouched` is used to add new waypoints between two existing waypoints, so it only fires if `addWaypoints` is enabled.

Of course, one could argue that the event should fire either way, but the handler to add a waypoint should only be added if the option is set - I think that makes more sense, and it should be fixed.

**Comment by Polymathronic** [comment]
Thanks for the clarification. In my scenario, I wanted to be able hide the routing table when the user lays down their finger on the map or some other part of the UI, and reveal it when they tap on the line.

**Comment by zeeuss** [comment]
Hi Polymathronic! How did you solve it?
